### PR TITLE
PAY-1562 fix more integration tests

### PR DESCRIPTION
**Pull request by mark-h-humi**
https://gethumi.atlassian.net/browse/PAY-1562

### CODE REVIEW

File: spec/integration/taxman2024/bug_8143_spec.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@
 
   let(:t) do
     Taxman2024::PersonalTaxDeductionsInput.new(
-      federal_personal_amount: nil,
```
Comment (claus-humi) [inline spec/integration/taxman2024/bug_8143_spec.rb:34]: Default is not used with Nil?

File: spec/integration/taxman2024/pay_4440_spec.rb [hunk 2]
```diff
@@ -88,6 +95,7 @@
     end
 
     it "matches the bonus t3" do
+      pending "not in PDOC"
```
Comment (claus-humi) [inline spec/integration/taxman2024/pay_4440_spec.rb:98]: These are from the spreadsheet I believe

**Review by claus-humi — COMMENTED** [review]
Just a few comments

File: spec/integration/taxman2024/pay_4440_spec.rb (hunk 2 above)
Comment (mark-h-humi) [reply spec/integration/taxman2024/pay_4440_spec.rb:98]: which spreadsheet?

File: spec/integration/taxman2024/bug_8143_spec.rb (hunk 1 above)
Comment (mark-h-humi) [reply spec/integration/taxman2024/bug_8143_spec.rb]: it is, fair enough.  fixed.

File: spec/integration/taxman2024/pay_4440_spec.rb (hunk 2 above)
Comment (mark-h-humi) [reply spec/integration/taxman2024/pay_4440_spec.rb:98]: anyhow maybe we can merge this one to get the newly passing tests and i'll make another pass on the pending tests

Comment (markjconlon-humi) [reply spec/integration/taxman2024/pay_4440_spec.rb:98]: works for me

**Review by markjconlon-humi — APPROVED** [review]

File: spec/integration/taxman2024/pay_4440_spec.rb (hunk 2 above)
Comment (mark-h-humi) [reply spec/integration/taxman2024/pay_4440_spec.rb:98]: will make a ticket for the remaining pending tests, thanks @markjconlon-humi

Comment (claus-humi) [reply spec/integration/taxman2024/pay_4440_spec.rb:98]: @mark-h-humi The implementation spreadsheet. T3 would appear on the implementation spreadsheet, and the original tests have been run through PDOC and the implementation spreadsheet to verify both intermediate and final results.

**Pull request merged**